<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableOptions</class>
 <widget class="QDialog" name="TableOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>10</horstretch>
    <verstretch>8</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>657</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>657</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>296</width>
     <height>270</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QSpinBox" name="font_size">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string extracomment="Font Size">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Font Size&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="Horiz_headers">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>Alphabetical</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="maxVisibleItems">
       <number>4</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Alphabetical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Numerical</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="Vert_headers">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="maxVisibleItems">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>Alphabetical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Numerical</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFontComboBox" name="fontComboBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
      <property name="placeholderText">
       <string extracomment="Select Font"/>
      </property>
      <property name="writingSystem">
       <enum>QFontDatabase::Latin</enum>
      </property>
      <property name="fontFilters">
       <set>QFontComboBox::ScalableFonts</set>
      </property>
      <property name="currentFont">
       <font>
        <family>Arial</family>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QSpinBox" name="row_count">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>702</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="sheet_name">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QSpinBox" name="column_count">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="LinkVisited">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>199</red>
            <green>216</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="LinkVisited">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>199</red>
            <green>216</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="LinkVisited">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>199</red>
            <green>216</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>702</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="continue_button">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_picture">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>261</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(:/Images/testimage.jpg)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Horiz_headers</tabstop>
  <tabstop>Vert_headers</tabstop>
  <tabstop>fontComboBox</tabstop>
  <tabstop>font_size</tabstop>
  <tabstop>sheet_name</tabstop>
  <tabstop>column_count</tabstop>
  <tabstop>row_count</tabstop>
  <tabstop>continue_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
